### PR TITLE
Redirect to library page when viewing self [PAR-98]

### PR DESCRIPTION
**Pull request by lemniscate8**
Will prevent friending yourself and requesting books from yourself.

### CODE REVIEW

File: src/FriendLibraryPage.tsx [hunk 1]
```diff
@@ -122,40 +122,51 @@ const FriendLibraryPage: React.FC = () => {
...
-        onSearchChange={filterResults}
-        header="Search this Library"
-      />
-      <AutoTable
-        data={filteredBooks}
-        title={<h3>Books</h3>}
-        placeholder={
-          books ? (
-            <span>No search results found</span>
+  if (auth.credential.userId?.toString() === id) {
```
Comment (lemniscate8) [inline src/FriendLibraryPage.tsx]: @Hashiphoto It looks like the userId is coming in as a number and not a string despite being typed as a string.

Comment (Hashiphoto) [reply src/FriendLibraryPage.tsx]: Oh. I can fix that in https://github.com/ParaLibrary/paralibrary-backend/pull/33 if you want?

**Review by Hashiphoto — APPROVED** [review]
Looks good!

File: src/FriendLibraryPage.tsx (hunk 1 above)
Comment (lemniscate8) [reply src/FriendLibraryPage.tsx]: Sure!

Comment (Hashiphoto) [reply src/FriendLibraryPage.tsx]: It has been fixed. I confirmed it sends the userId as a string on login. I've gone ahead and merged https://github.com/ParaLibrary/paralibrary-backend/pull/33

**Pull request merged**